introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/inference-benchmarking-custom-probe-example.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "input_512_output_128": create_test_payload(input_words=512, output_tokens=128),
</pre><p>Have we thought through whether using the same payload captures the benchmarking correctly? For instance if the number of tokens vary even a little bit (120, 130), can the inference latency be significantly different due to padding tokens or otherwise?</p><p> </p><p>Also, this is probably obvious/given but we should establish that any optimization process done at the container does not impact model output i.e. even at high concurrent requests with different size payload, model produces the same output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>